### ISSUE TITLE
Change the icon

### ISSUE THREAD

**Issue by gagarine** [belaviyo/keepass-macpass-helper#13]
The icon is a "lock" but the action has nothings to do with locking somethings.

I think using a key instead of lock for the icon would be more intuitive. Or somethings that represent "accounts" but this is harder to make it clear I think.

**Comment by belaviyo** [comment]
True, if somebody is willing to design a new icon, we can consider switching. I have to say that users are used to this look though!

**Comment by gamedevsam** [comment]
I don't see a reason for this. I like the current icon just fine myself. Recommend closing this issue.
